LiskHQ/lisk-sdk
# Merge release 2.0.0 into development may 24

### What was the problem?

Release/2.0.0 was not merged into development

### How did I fix it?

By merging and solving conflicts

### How to test it?

- Build should be green
- Solved conflicts in:
- CONFLICT (modify/delete): framework/src/modules/chain/submodules/transport.js deleted in HEAD and modified in release/2.0.0. Version release/2.0.0 of framework/src/modules/chain/submodules/transport.js left in tree.
- CONFLICT (modify/delete): framework/src/modules/chain/submodules/delegates.js deleted in HEAD and modified in release/2.0.0. Version release/2.0.0 of framework/src/modules/chain/submodules/delegates.js left in tree.
- CONFLICT (content): Merge conflict in framework/src/modules/chain/logic/block.js
- CONFLICT (content): Merge conflict in framework/src/modules/chain/loader.js
- CONFLICT (content): Merge conflict in framework/src/controller/controller.js

### Review checklist

* The PR resolves NO ISSUE MERGE OF FEATURE BRANCHES
* All new code is covered with unit tests
* All new code was formatted with Prettier
* Linting passes
* Tests pass
* Commit messages follow the [commit guidelines](CONTRIBUTING.md#git-commit-messages)
* Documentation has been added/updated
